code-experiments/build/java/ExampleExperiment.java

```diff
@@ -71,7 +75,7 @@ public static void main(String[] args) {
          * Adapt to your need. Note that the experiment is run according
          * to the settings, defined in exampleExperiment(...) below.
          */
-		exampleExperiment("bbob", "bbob", randomGenerator);
```

Comment:
We should keep the same functionality everywhere: hence, I would opt for going back to `exampleExperiment("bbob", "bbob", randomGenerator);`. The new `bbob-constrained` call should appear though in the comment (together with how to call the `bbob-biobj` suite probably).